test/plugins/wasmedge_stablediffusion/wasmedge_stablediffusion.cpp

```diff
@@ -0,0 +1,322 @@
+                                                    PromptData.size(),
+                                                    SessionId,
+                                                    0,
+                                                    0,
+                                                    0,
+                                                    0,
+                                                    64,
+                                                    64,
+                                                    -1,
+                                                    7.0f,
+                                                    0,
+                                                    20,
+                                                    42,
+                                                    1,
+                                                    0.90f,
+                                                    20.0f,
+                                                    0,
+                                                    0,
+                                                    0,
+                                                    0,
+                                                    0,
+                                                    0,
+                                                    0,
+                                                    OutputPathPtr,
+                                                    OutputPath.size(),
+                                                    OutputPtr,
+                                                    65532,
+                                                    BytesWrittenPtr},
+        Errno));
+    EXPECT_EQ(Errno[0].get<int32_t>(), static_cast<uint32_t>(ErrNo::Success));
+    auto BytesWritten = *MemInst.getPointer<uint32_t *>(BytesWrittenPtr);
+    EXPECT_GE(BytesWritten, 50);
+    std::ifstream Fin(OutputPath.data(), std::ios::in | std::ios::binary);
+    EXPECT_FALSE(Fin.fail());
+    Fin.close();
+  }
+  writeBinaries<char>(MemInst, ModelPath, ModelPathPtr);
+  writeBinaries<char>(MemInst, QuantModelPath, QuantModelPathPtr);
+  // Test: create_context -- create context for image to image.
+  {
+    EXPECT_TRUE(HostFuncCreateContext.run(
+        CallFrame,
+        std::initializer_list<WasmEdge::ValVariant>{
+            QuantModelPathPtr,
+            static_cast<uint32_t>(QuantModelPath.size()),
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+            -1,
+            31,
+            1,
+            0,
+            0,
+            0,
+            0,
+            SessionPtr}, // NThreads=-1,
+                         //  wtype=31(SD_TYPE_COUNT), RngType=CUDA_RNG,
+                         //  Schedule=DEFAULT, Other is false
+        Errno));
```

Comment:
I think a better approach is comment beside each argument, so ppl no need to compare.